Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -373,8 +373,8 @@ def main():
     try:
         if command == 'fetch-incidents':
             incidents, new_last_run = fetch_incidents_command(client,
-                                                              fetch_time=params.get('fetchTime'),
-                                                              fetch_limit=params.get('fetchLimit'),
+                                                              fetch_time=params.get('first_fetch'),
+                                                              fetch_limit=params.get('max_fetch'),
```

Comment:
```suggestion
                                                              fetch_time=params.get('FetchTime, first_fetch'),
                                                              fetch_limit=params.get('fetchLimit, max_fetch'),
```
+ a comment why the next dev should not cahnge this line(due to BC)
this will be have to be done to all the .py files in this PR.